src/aspire/utils/matlab_compat.py

```diff
@@ -9,26 +9,6 @@
 import scipy.sparse as sparse
 
 
-def m_reshape(x, new_shape):
-    # This is a somewhat round-about way of saying:
-    #   return x.reshape(new_shape, order='F')
-    # We follow this approach since numba/cupy don't support the 'order'
-    # argument, and we may want to use those decorators in the future
-    # Note that flattening is required before reshaping, because
```

Comment:
And now we'll never know why…

Reply:
🤣

Reply:
It was only true for Numba, which we didn't use....